File: Iniciante/1158 - Soma de Ímpares Consecutivos III/JavaScript/1158.js
const input = require('fs').readFileSync(0, 'utf-8').trim().split(/\s+/).map(Number);

let index = 0;
const n = input[index++];

for (let i = 0; i < n; i++) {
    let x = input[index++];
    let y = input[index++];

    let sumTotal = 0;
    let count = y;
    let value = x;

    if (value % 2 === 0) {
        value++;
    }

    while (count > 0) {
        sumTotal += value;
        value += 2;
        count--;
    }

    console.log(sumTotal);
}

File: Iniciante/1158 - Soma de Ímpares Consecutivos III/Python/1158.py
def main():
    n = int(input())

    for _ in range(n):
        x, y = map(int, input().split())

        sum_total = 0
        count = y
        value = x

        if value % 2 == 0:
            value += 1

        while count > 0:
            sum_total += value
            value += 2
            count -= 1

        print(sum_total)


if __name__ == "__main__":
    main()
